### ISSUE TITLE
CMS: wrong GT for analysis in the 2012 RunB and RunC records

### ISSUE THREAD

**Issue by katilp** [cernopendata/opendata.cern.ch#2790]
The CMS primary records have wrong Global Tag in system details

```
"system_details": {
      "global_tag": "FT_53_LV5_AN1",
      "release": "CMSSW_5_3_32"
    },
```

NB also RAW
It should be FT53_V21A_AN6

**Comment by tiborsimko** [comment]
@ANovorolnik @Filip1509 Please have a look at #2792 and finish the necessary corrections in analogous manner

**Comment by Filip1509** [comment]
@katilp I have seen occurences of wrong GT in eventdisplay files, should I change them as well?

cernopendata/modules/fixtures/data/records/cms-eventdisplay-files-Run2012B.json:      "global_tag": "FT_53_LV5_AN1",
cernopendata/modules/fixtures/data/records/cms-eventdisplay-files-Run2012C.json:      "global_tag": "FT_53_LV5_AN1",

**Comment by katilp** [comment]
@Filip1509 Thanks for pointing that, it is indeed wrong there as well. Very useful, and good work! 

Looking at it now, I realize that these fields (i,e, "System details": the CMSSW version for analysis, and the GT) would actually not be needed in the event display files, because the CMSSW software package is not needed when looking at these data, they are opened in the event display.

So either they are corrected (even if not useful), or removed.
@tiborsimko What would you think?

**Comment by tiborsimko** [comment]
@katilp @Filip1509 Yes, I also think we should rather remove that information from event display files, i.e. if the file format is not root, we can remove the whole `system_details` JSON field.

**Comment by tiborsimko** [comment]
Merged #2792 and #2794; the removal of `system_details` will be done by @Filip1509 separately.

**Comment by tiborsimko** [comment]
... and completed by #2793 and #2795.
